File: auxiliatura/POO/1/Persona.py
class Persona:
    def __init__(self, nombre, edad, ciudad):
        self.nombre = nombre
        self.edad = edad
        self.ciudad = ciudad

    def mostrar(self):
        print(f"Hola, soy {self.nombre} de {self.ciudad}")

    def mayor(self):
        if self.edad < 18 and self.edad >= 0:
            print("Es menor de edad")
        elif self.edad >= 18:
            print("Es mayor de edad")
        else:
            print("Edad inválida")

if __name__ == "__main__":
    p1 = Persona("Jhon", 11, "La Paz")
    p2 = Persona("Efrain", 18, "Cochabamba")
    p1.mostrar()
    p1.mayor()
    p2.mostrar()
    p2.mayor()

File: auxiliatura/POO/1/Persona.java
package persona;

public class Persona {
    private String nombre;
    private int edad;
    private String ciudad;
    
    public Persona(String nombre, int edad, String ciudad){
        this.nombre = nombre;
        this.edad = edad;
        this.ciudad = ciudad;
    }

    public void mostrar(){
        System.out.println("Hola, soy " + nombre + " de " + ciudad);
    }

    public void mayor(){
        if (edad < 18 && edad >= 0)
            System.out.println("Es menor de edad");
        else if (edad >= 18)
            System.out.println("Es mayor de edad");
        else
            System.out.println("Edad inválida");
    }

    public static void main(String[] args) {
        Persona p1 = new Persona("Jhon", 11, "La Paz");
        Persona p2 = new Persona("Efrain", 18, "Cochabamba");
        p1.mostrar();
        p1.mayor();
        p2.mostrar();
        p2.mayor();
        
    }
}
